pkg/policy/mapstate.go

```diff
@@ -65,6 +65,7 @@ type MapState interface {
 	AddVisibilityKeys(PolicyOwner, uint16, *VisibilityMetadata, ChangeState)
 	Len() int
 	Equals(MapState) bool
+	Diff(expected MapState) string
```

Comment:
suggestion: If it's just for use in testing, can we explicitly take a `testing.T` argument (even if we don't need it) to ensure it's not used elsewhere?

Reply:
good point, will do!

Reply:
Added!